stglib/exo.py

```diff
@@ -577,210 +578,26 @@ def exo_qaqc(ds):
-            "Values filled where percent difference between "
-            "%d-point median filter and original values is greater "
-            "than %f. " % (kernel_size, ds.attrs[var + "_med_diff_pct"])
-        )
-
-        ds = utils.insert_note(ds, var, notetxt)
-
-    return ds
-
-
-def trim_bad_ens(ds, var):
-    if var + "_bad_ens" in ds.attrs:
-        inc = np.arange(0, len(ds.attrs[var + "_bad_ens"]), 2)
-        for n in inc:
-            print(
-                "%s: Trimming using bad_ens %s"
-                % (var, str(ds.attrs[var + "_bad_ens"][n : n + 2]))
-            )
-            if isinstance(ds.attrs[var + "_bad_ens"][n], str):
-                bads = (ds["time"] >= np.datetime64(ds.attrs[var + "_bad_ens"][n])) & (
-                    ds["time"] <= np.datetime64(ds.attrs[var + "_bad_ens"][n + 1])
-                )
-                ds[var] = ds[var].where(~bads)
-            else:
-                bads = np.full(ds[var].shape, False)
-                bads[
-                    np.arange(
-                        ds.attrs[var + "_bad_ens"][n], ds.attrs[var + "_bad_ens"][n + 1]
-                    )
-                ] = True
-                ds[var][bads] = np.nan
-
-            notetxt = "Data clipped using bad_ens values of %s. " % (
-                str(ds.attrs[var + "_bad_ens"])
-            )
-
-            ds = utils.insert_note(ds, var, notetxt)
-
-    return ds
-
-
-def trim_by_salinity(ds, var):
-    if (
-        "trim_by_salinity" in ds.attrs
-        and ds.attrs["trim_by_salinity"].lower() == "true"
-        and var in ds
-    ):  # xarray doesn't support writing attributes as booleans
-        if (
-            "trim_by_salinity_exclude" in ds.attrs
-            and var in ds.attrs["trim_by_salinity_exclude"]
-        ):
-            pass
-        else:
-            print("%s: Trimming using valid salinity threshold" % var)
-            ds[var][ds["S_41"].isnull()] = np.nan
-
-            if var != "S_41":
-                notetxt = "Values filled using valid salinity threshold. "
-
-                ds = utils.insert_note(ds, var, notetxt)
-
-    return ds
-
-
-def trim_by_any(ds, var):
-    attrlist = []
-    for a in ds.attrs:
-        if "trim_by" in a:
-            attrlist.append(a)
-
-    for a in attrlist:
-        if (
-            a in ds.attrs and ds.attrs[a].lower() == "true" and var in ds
-        ):  # xarray doesn't support writing attributes as booleans
-            if f"{a}_exclude" in ds.attrs and var in ds.attrs[f"{a}_exclude"]:
-                pass
-            else:
-                trimvar = a.split("trim_by_")[-1]
-                print(f"{var}: Trimming using valid {trimvar} threshold")
-                ds[var][ds[trimvar].isnull()] = np.nan
-
-                if var != trimvar:
-                    notetxt = f"Values filled using valid {trimvar} threshold. "
-
-                    ds = utils.insert_note(ds, var, notetxt)
-
-    return ds
+# remove qaqc defs from exo.py
```

Comment:
This comment is helpful for review but should probably be removed

Reply:
removed.